src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -200,12 +202,17 @@ const DetailsMetrics = ({ selectedItem }) => {
   }, [handleWindowScroll])
 
   useEffect(() => {
-    dispatch(
-      detailsActions.fetchModelEndpointMetrics(
-        selectedItem.metadata.project,
-        selectedItem.metadata.uid
+    const getModelEndpointMetrics = async () => {
+      await dispatch(
+        detailsActions.fetchModelEndpointMetrics(
+          selectedItem.metadata.project,
+          selectedItem.metadata.uid
+        )
       )
-    )
+      setIsAllowedFetchModelEndpointMetrics(true)
+    }
+
+    getModelEndpointMetrics()
```

Comment:
just use `then` on the promise instead of creating a new function

Reply:
done.